i18n/translations/en/response-api.json

```diff
@@ -86,7 +86,9 @@
       "html": {
         "label": "Individual copies (Separate HTML files)"
       }
-    }
+    },
+    "previewCsvStructure": "Preview of the file structure for CSV responses:",
```

Comment:
@anikbrazeau 

Fresh strings if you want to edit at all

Reply:
LGTM!